lib/src/configuration.dart

```diff
@@ -99,6 +99,11 @@ class Configuration {
   /// The FIFO special files are very lightweight and the main [Realm] file will still be stored in the location defined
   /// by the [path] you  property. This property is ignored if the directory defined by [path] allow FIFO special files.
   final String? fifoFilesFallbackPath;
+
+  /// Specifies if a [Realm] file format should be automatically upgraded
+  /// if it was created with an older version of the [Realm] library.
+  /// If set to true and a file format upgrade is required, an error will be thrown instead.
```

Comment:
```suggestion
  /// An exception will be thrown if a file format upgrade is required.
```

Reply:
One note: We should be careful when wording `exception` and `error` since these have different meaning.